auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -178,11 +224,43 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return this.headers?.filter(x => x.show)
   }
 
+  private get getStatuses (): string[] {
+    const items = []
+    for (const i in this.filteredItems) {
+      if (!items.includes(this.status(this.filteredItems[i]))) {
+        items.push(this.status(this.filteredItems[i]))
+      }
+    }
+    return items
+  }
+
+  private get getTypes (): string[] {
+    const items = []
+    for (const i in this.filteredItems) {
+      if (!items.includes(this.type(this.filteredItems[i]))) {
```

Comment:
These look fairly similar, might be worth refactoring?

Reply:
sure yeah I could just pass in the fn